manageiq-operator/cmd/main.go

```diff
@@ -30,9 +30,11 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	ctrl "sigs.k8s.io/controller-runtime"
+	ctrlcache "sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	ctrlmetrics "sigs.k8s.io/controller-runtime/pkg/metrics/server"
```

Comment:
Not that it really matter, but I would expect this to be next to the other new one sort of alphabetically?

Reply:
Yeah, me too. Unfortunately the tooling keeps moving it down there 🤦🏻‍♂️